### ISSUE TITLE
[Keyboard navigation-Help-Projects] Page Title is not descriptive enough to the users

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2537]
**User Experience:**
If the title of a page or document does not  provide information specific to the currently viewed page or document. Then it might create confusion among screen reader dependent or it might misguide them

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to https://makecode.com/
2-Navigate through all controls (say computing education, about Microsoft makecode, get inspired, resources elements etc)lying on the page.
3-Navigate to the 'Get inspired' and select it.
4-Verify, If page title is changed according to selection or not.

**Actual Result**
Page title is not descriptive enough to the users. It is more generic LIke :- Microsoft MakeCode instead of being precise and clear Like:- Microsoft MakeCode-Get Inspired.

**Expected Result**
Page title should be descriptive enough to the users.

**Note:** This issue is through out the Application

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/Forms/AllItems.aspx?id=%2Fteams%2Fmsenable%2FDocuments%2FLCA%2FMAS%20%2D%20July%201%2C%202017%20Release%2FMAS%202%2E4%2E2%20%E2%80%93%20Page%20Titled%2Edocx&parent=%2Fteams%2Fmsenable%2FDocuments%2FLCA%2FMAS%20%2D%20July%201%2C%202017%20Release

**Suggested fix:**
All HTML and XHTML documents, including those in individual frames in a frameset, must have a title element in the head section that defines in a simple phrase the purpose of the document. This helps users to orient themselves within the site quickly without having to search for orientation information in the body of the page.  
 
Note: The (mandatory) title element, which only appears once in a document, is different from the title attribute, which may be applied to almost every HTML and XHTML element. 
 e.g.
<html xmlns="http://www.w3.org/1999/xhtml">    
   <head>      
      <title>The World Wide Web Consortium</title>      
   </head>    
   <body>      
      ...    
   </body>  
</html>   

**Please Refer attachment for more details**
![mas2 4 2_home page_same title is defined for all the pages](https://user-images.githubusercontent.com/30434417/28819752-d76feb82-76cc-11e7-8ea9-160aae4bcf35.jpg)

**Comment by microsoftsam** [comment]
Fixed

**Comment by v-mesala** [comment]
Bug is verified on URL :https://makecode.com/ and bug is fixed .Hence clsoed the bug and added the closed tag.
